cards/src/main/java/com/handonn/finapp/cards/service/ICardService.java

```diff
@@ -0,0 +1,20 @@
+package com.handonn.finapp.cards.service;
+
+
+import com.handonn.finapp.cards.model.CardCriteria;
+import com.handonn.finapp.cards.model.CardDto;
+import org.springframework.data.domain.Page;
+
+public interface ICardService {
+
+    void create(CardDto cardDto);
+
+    Page<CardDto> getAllCard(CardCriteria filter);
```

Comment:
Change **filter** to **criteria** 
Because we will works with Spring Security Filter later, it mays be misleading